portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,6 +38,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String comment = request.getParameter("comment-input");
     commentsList.add(comment);
 
+    Entity commentEntity = new Entity("stringComment");
+    commentEntity.setProperty("comment", comment);
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
nit - consider moving this to be an member variable of the class. It won't matter much for this instance, but in the future when you'll be reading and writing from the datastore it will be nice to have a single reference to it instead of having to get one every time.

Reply:
that sounds like a good plan. ive been trying to find some sources on how to properly write member variables, as im not familiar with it, but im not sure how to put these lines into a member variable of the class. do you know of some documentation somewhere that can help me? theres some help on cloud.google.com, but it doesnt say on how to reference it in a member variable.

Reply:
https://cloud.google.com/appengine/docs/standard/java/datastore/creating-entities
^^ this is what i've been using as a guide